portality/settings.py

```diff
@@ -900,6 +920,52 @@
     "private_source": "portality.lib.query_filters.private_source",
     "public_source": "portality.lib.query_filters.public_source",
 }
+# Exclude the fields that doesn't want to be searched by public queries
+# This is part of non_public_fields_validator.
+EXCLUDED_FIELDS = [
+    "admin.notes.note",
+    "admin.notes.id",
+    "admin.notes.author_id"
+]
+
+ADMIN_NOTES_SEARCH_MAPPING = {
```

Comment:
In the `settings.py` file can we group this with the other MAPPINGS keys, a bit above in the file

Reply:
As per my understanding, above mappings are general mappings which apply for all. ADMIN_NOTES_SEARCH_MAPPING applies only for application and journal. So I have separated them. Please correct me if wrong.